Mathlib/CategoryTheory/GradedObject/Unitor.lean

```diff
@@ -125,7 +125,7 @@ lemma mapBifunctorLeftUnitor_inv_naturality :
     ι_mapBifunctorMapMap]
   dsimp
   rw [Functor.map_id, NatTrans.id_app, id_comp]
-  erw [← NatTrans.naturality_assoc, ← NatTrans.naturality_assoc]
+  rw [← NatTrans.naturality_assoc, ← NatTrans.naturality_assoc]
```

Comment:
Can this be combined with the previous line?

Reply:
Yep!